### PR TITLE
add: files & styles

### PR DESCRIPTION
**Pull request by shilpe26**
Header & Footer of BookHub website with features like:-

▪️Added Navbar
▪️Added Footer
▪️Added Routes that connect to the login page, signup page
▪️Added Mockman route

Screenshot (336)

### CODE REVIEW

File: src/styles/navbar.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    border: 2px solid var(--secondary-gray);
+    border-radius: 6px;
+    cursor: pointer;
+}
+
+.heading span:hover {
+    border-color: var(--yellow-color);
+}
+
+.btn {
```
Comment (anshulraheja) [inline src/styles/navbar.css:46]: You could have used a more specific class name here. 
Rest everything is good to merge

**Review by anshulraheja — APPROVED** [review]

File: src/styles/navbar.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    position: fixed;
+    left: 0;
+    top: 0;
+    right: 0;
+    background-color: var(--bg-color);
+    padding: 1.5rem 4%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 999;
```
Comment (mansipadgelwar) [inline src/styles/navbar.css:72]: On my last PR review, someone told me not to give this much z-index value. You can change it if you want rest looks good to me.

**Review by mansipadgelwar — APPROVED** [review]

File: src/styles/navbar.css (hunk 1 above)
Comment (shilpe26) [reply src/styles/navbar.css:46]: okay

File: src/styles/navbar.css (hunk 2 above)
Comment (shilpe26) [reply src/styles/navbar.css:72]: okay...done

**Pull request merged**